error.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <NuxtLayout>
```

Comment:
```suggestion
  <NuxtLayout name="default">
```

You miss the `name="default"`.